Terrain world components documentation (#1242)
o3de/o3de.org: content/docs/user-guide/components/reference/terrain/world-renderer.md

---
linktitle: Terrain World Renderer
title: Terrain World Renderer Component

> description: The Terrain World Renderer component renders the current world.

Reviewer: ```suggestion
description: The Terrain World Renderer component renders terrain within the world.
```

Author: Changed.

toc: true
---

> The **Terrain World Renderer** component renders the current world.

Reviewer: ```suggestion
The **Terrain World Renderer** component renders terrain within the bounds set by the **Terrain World** component.
```

Author: Much better. Changed.

## Provider ##

[Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)

## Dependencies ##

[Terrain World](/docs/user-guide/components/reference/terrain/world)

## Properties ##

> ![\[Terrain World Renderer component interface.\]](/images/user-guide/component/terrain/terrain/ui-terrain-world-renderer-A.png)

Reviewer: ```suggestion
![Terrain World Renderer component interface.](/images/user-guide/component/terrain/terrain/ui-terrain-world-renderer-A.png)
```

Author: Changed.
